mypy_django_plugin/lib/helpers.py

```diff
@@ -34,7 +34,7 @@
     MethodContext,
     SemanticAnalyzerPluginInterface,
 )
-from mypy.plugins.common import add_method
+from mypy.plugins.common import add_method_to_class
```

Comment:
Justification of this change: `add_method` is mentioned as deprecated by mypy https://github.com/python/mypy/blob/2bfa3b48e671f461b9d9ad9e49dcb83c31bade37/mypy/plugins/common.py#L85-L96

Might make more sense to perhaps change that in a different PR